homeassistant/components/rflink/sensor.py

```diff
@@ -35,9 +43,68 @@
 SENSOR_ICONS = {
     "humidity": "mdi:water-percent",
     "battery": "mdi:battery",
-    "temperature": "mdi:thermometer",
 }
 
+SENSOR_TYPES = (
+    # check new descriptors against PACKET_FIELDS & UNITS from rflink.parser
+    SensorEntityDescription(
+        key="distance",
+        name="Distance",
+        device_class=SensorDeviceClass.DISTANCE,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    SensorEntityDescription(
+        key="barometric_pressure",
+        name="Barometric pressure",
+        device_class=SensorDeviceClass.PRESSURE,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    SensorEntityDescription(
+        key="average_windspeed",
+        name="Average windspeed",
+        device_class=SensorDeviceClass.SPEED,
+        state_class=SensorStateClass.MEASUREMENT,
```

Comment:
While we where working a new wind speed device class was added.

Reply:
Changed here and the other ones